pages/text-diff.tsx

```diff
@@ -66,13 +144,48 @@ export default function TextDiffPage() {
 
   return (
     <Layout title='Text Difference'>
-      <Heading>Text Diff</Heading>
+      <Heading>Text Diff {selectedOptions?.label}</Heading>
       <Typography
         paragraph
         textAlign='center'
       >
         Type or paste text into both fields to check the difference.
       </Typography>
+      <Box
+        display='flex'
+        flexDirection='column'
+        justifyContent='stretch'
+        width={1000}
+        maxWidth='100%'
+        gap={2}
+        mb={3}
+      >
+        <FormControl>
+          <InputLabel id='diff-select-label'>
+            Choose Diff Options
```

Comment:
This is what you should use in `getByLabelText` ("Choose Diff Options")

As a last resort you can add `data-testid="some-unique-string"` to any element, but it is better to use the other queries (e.g. `getByLabelText`) when possible